core/ffmpeg/transcoder.go

```diff
@@ -125,7 +131,7 @@ func (t *Transcoder) getString() string {
 		// Filenames
 		"-master_pl_name", "stream.m3u8",
 		"-strftime 1",                                                               // Support the use of strftime in filenames
-		"-hls_segment_filename", path.Join(t.segmentOutputPath, "/%v/stream-%s.ts"), // Each segment's filename
+		"-hls_segment_filename", path.Join(t.segmentOutputPath, "/%v/stream-%s-"+string(t.segmentIdentifier)+".ts"), // Each segment's filename
```

Comment:
Why is `string(t.segmentIdentifier)` required if `t.segmentIdentifier` is already a string?

Reply:
oops! initially used it with a direct function call. will remove that.